SpellCheck/pom.xml

```diff
@@ -25,43 +17,41 @@
 			<comments>Educational Online Test Delivery System Copyright (c) 2013 American Institutes for Research</comments>
 		</license>
 	</licenses>
+
 	<dependencies>
+		<dependency>
+			<groupId>com.atlascopco</groupId>
+			<artifactId>hunspell-bridj</artifactId>
+			<version>1.0.4</version>
+		</dependency>
+
 		<!-- Start Dependency shared project -->
 		<dependency>
 			<groupId>org.opentestsystem.shared</groupId>
 			<artifactId>shared-web</artifactId>
 		</dependency>
+
 		<dependency>
 			<groupId>javax.servlet</groupId>
 			<artifactId>servlet-api</artifactId>
 			<scope>provided</scope>
 		</dependency>
-		<dependency>
-	        <groupId>dk.dren</groupId>
-			<artifactId>hunspell</artifactId>
-			<version>1.3.2-SNAPSHOT</version>
-     	 </dependency>
-     	 <dependency>
-			<groupId>com.sun.jna</groupId>
-			<artifactId>jna</artifactId>
-			<version>3.0.9</version>
-		</dependency>
+
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<version>4.12</version>
```

Comment:
I moved these version declarations into the parent pom and shared them among all the submodules in TDS_Student.